app/serializers/theme_serializer.rb

```diff
@@ -47,6 +48,14 @@ def include_theme_field_values?
     @include_theme_field_values || object.remote_theme_id.nil?
   end
 
+  def screenshot_url
+    object
+      .theme_fields
+      .find { |field| field.type_id == ThemeField.types[:theme_screenshot_upload_var] }
+      &.upload
+      &.url
+  end
```

Comment:
Minor one, but might be worth having a `ThemeField#upload_url` delegator.

Reply:
Yes good point, will add :+1: